ProcessMaker/Models/User.php

```diff
@@ -135,8 +135,16 @@ public static function rules($existing = null)
         $unique = Rule::unique('users')->ignore($existing);
 
         $checkUserIsDeleted = function($attribute, $value, $fail) use ($existing) {
-            if (!$existing && User::where($attribute, $value)->exists()) {
-                $fail('userExists');
+            if (!$existing) {
+                $user = User::withTrashed()->where($attribute, $value)->first();
```

Comment:
When adding a non-deleted user the 'Deleted User Found' modal appears. 
Changing `withTrashed()` to `onlyTrashed()` appears to fix the issue.

`withTrashed()`: returns all records including trashed

`onlyTrashed()`: returns only the trashed records

https://laravel.com/docs/6.x/scout#soft-deleting